explore/src/main/scala/explore/visualization/OffsetSVG.scala

```diff
@@ -16,6 +16,9 @@ import react.common.Style
 import react.floatingui.*
 import react.floatingui.hooks.*
 
+enum OffsetType:
+  case Science, Acquisition
```

Comment:
Can we use `lucuma.core.enums.SequenceType` here?

Reply:
Hmm, it is not exactly the same but the options match, I guess it can be argued that they are offsets for those sequence types. I'll make the change